DOCUMENTATION.md

```diff
@@ -107,94 +115,94 @@ const sdk = fromSharedOptions();
     * [.models](#balena.models) : <code>object</code>
         * [.application](#balena.models.application) : <code>object</code>
             * [.tags](#balena.models.application.tags) : <code>object</code>
-                * [.getAllByApplication(slugOrUuidOrId, [options])](#balena.models.application.tags.getAllByApplication) ⇒ <code>Promise</code>
+                * [.getAllByApplication(slugOrId, [options])](#balena.models.application.tags.getAllByApplication) ⇒ <code>Promise</code>
```

Comment:
I think that you need to run the full `npm run build`, since the docs are generated from the jsdocs found in the unminified js build output.
I don't get any diff locally on master after an `npm run build`.

Reply:
Ahh yes you are right